examples/PTI_simulation/PTI_Simulation_Recon3D.py

```diff
@@ -499,9 +499,9 @@
 
 ### select slices to plot ###
 spacing = 4
-z_layer = 50
-x_layer = 100
-y_layer = 100
+z_layer = L//2
+x_layer = N//2
+y_layer = M//2
```

Comment:
Formatting: missing spaces.

Reply:
Good eye. I wonder if I have a weird setting with my vscode black formatter...I've been getting the odd glitch like this (also adding and removing empty lines).